.stylelintrc

```diff
@@ -0,0 +1,147 @@
+{
+  "rules": {
+    "at-rule-empty-line-before": [ "always", {
+      except: [
+        "blockless-after-same-name-blockless",
+        "first-nested",
+      ],
+      ignore: ["after-comment"],
+    } ],
+    "at-rule-name-case": "lower",
+    "at-rule-name-space-after": "always-single-line",
+    "at-rule-no-vendor-prefix": true,
```

Comment:
Exactly, in fact we already have those in all of our projects. So adding them would do no good.